config/environments/development.rb

```diff
@@ -73,10 +78,10 @@
   config.action_mailer.smtp_settings = Rails.application.secrets[:smtp_settings]
 
   # Bullet
-  config.after_initialize do
-    Bullet.enable = true
-    Bullet.bullet_logger = true
-    Bullet.console = true
-    Bullet.add_footer = true
-  end
+  # config.after_initialize do
+  #   Bullet.enable = true
+  #   Bullet.bullet_logger = true
+  #   Bullet.console = true
+  #   Bullet.add_footer = true
+  # end
```

Comment:
Do we want to remove Bullet? If not, maybe open a ticket to check back on the Rails 6 issues.